scripts/exrrfs_ioda_prepbufr.sh

```diff
@@ -0,0 +1,213 @@
+#
+"ORION")
+  APRUN="srun"
+  ;;
+#
+esac
+#
+#-----------------------------------------------------------------------
+#
+# Extract from CDATE the starting year, month, day, and hour of the
+# forecast.  These are needed below for various operations.
+#
+#-----------------------------------------------------------------------
+#
+START_DATE=$(echo "${CDATE}" | sed 's/\([[:digit:]]\{2\}\)$/ \1/')
+YYYYMMDDHH=$(date +%Y%m%d%H -d "${START_DATE}")
+JJJ=$(date +%j -d "${START_DATE}")
+
+YYYY=${YYYYMMDDHH:0:4}
+MM=${YYYYMMDDHH:4:2}
+DD=${YYYYMMDDHH:6:2}
+HH=${YYYYMMDDHH:8:2}
+YYYYMMDD=${YYYYMMDDHH:0:8}
+
+YYJJJHH=$(date +"%y%j%H" -d "${START_DATE}")
+PREYYJJJHH=$(date +"%y%j%H" -d "${START_DATE} 1 hours ago")
+
+#
+#-----------------------------------------------------------------------
+#
+# Get into working directory
+#
+#-----------------------------------------------------------------------
+#
+print_info_msg "$VERBOSE" "
+Getting into working directory for PrepBUFR obseration process ..."
+#
+#-----------------------------------------------------------------------
+#
+# copy template yaml files
+#
+#-----------------------------------------------------------------------
+#
+cp ${PARM_IODACONV}/*.yaml .
+#
+#-----------------------------------------------------------------------
+#
+# link the executable file
+#
+#-----------------------------------------------------------------------
+#
+export pgm="bufr2ioda.x"
+. prep_step
+#
+#
+#-----------------------------------------------------------------------
+#
+# check the existence of the PrepBUFR file for the current cycle, 
+# if the file is present, convert the data into ioda format for 
+# aircraft, ascatw, gpsipw, mesonet, profiler, rassda,
+# satwnd, surface, upperair subsets.
+#
+#-----------------------------------------------------------------------
+#
+run_process_prepbufr=false
+obs_file=prepbufr
+checkfile=${OBSPATH}/${YYYYMMDDHH}.rap.t${HH}z.prepbufr.tm00
+if [ -r "${checkfile}" ]; then
+  print_info_msg "$VERBOSE" "Found ${checkfile}; Use it as observation "
+  cp -p ${checkfile} ${obs_file}
+  run_process_prepbufr=true
+else
+  print_info_msg "$VERBOSE" "Warning: PrepBUFR file for ${YYYYMMDDHH} does not exist!"
+fi
+
+#
+#-----------------------------------------------------------------------
+#
+# Modify yaml template and run the process
+# for converting prepbufr file 
+#
+#-----------------------------------------------------------------------
+#
+formatted_time=$(date -d"${YYYYMMDDHH:0:8} ${YYYYMMDDHH:8:2}" '+%Y-%m-%dT%H:%M:%SZ')
+
+for yamlfile in *.yaml; do
+  message_type=$(echo "$yamlfile" | cut -d'_' -f4 | sed 's/\..*//')
+
+  if grep -q "referenceTime" ${yamlfile}; then
+    sed -i "s/referenceTime:.*/referenceTime: ${formatted_time}/" ${yamlfile}
+  fi
+
+  if [[ ${run_process_prepbufr} ]]; then
+    $APRUN ${EXECdir}/$pgm ${yamlfile} >> $pgmout 2>errfile
+    export err=$?; err_chk
+    cp stdout_prepbufr_${message_type} $comout/stdout.t${HH}z.prepbufr_${message_type}
```

Comment:
Since the name of the log file is changed to `pgmout` above, the `stdout_prepbufr_` will not be generated.